bayesian/api/api_v2.py

```diff
@@ -160,6 +164,14 @@ def get(ecosystem, package, version):
             metrics_payload.update({"status_code": 200, "value": time.time() - st})
             _session.post(url=METRICS_SERVICE_URL + "/api/v1/prometheus", json=metrics_payload)
             return analyses_result
+        elif os.environ.get("SKIP_UNKNOWN_PACKAGE_FLOW", "") == "1":
+            msg = f"No data found for {ecosystem} package {package}/{version} " \
+                   "and skip unknown package flow is enabled"
+
+            metrics_payload.update({"status_code": 404, "value": time.time() - st})
```

Comment:
adding a 404 here doesnt look correct to me. Also it will be tracked on prometheus. We are purposefully stopping ingestion so i dont think we should besending 404, rather it should be 202